include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -328,6 +328,15 @@ indexstoredb_index_symbols_contained_in_file_path(_Nonnull indexstoredb_index_t
                                                   const char *_Nonnull path,
                                                   _Nonnull indexstoredb_symbol_receiver_t);
 
+/// Iterates over all the symbol occurrences contained in \p path
```

Comment:
Let’s clarify that `path` is a source file path, not eg. a path to a unit file or a record file

```suggestion
/// Iterates over all the symbol occurrences contained in the source file at \p path
```